libs/node-server-sdk/src/lib/configuration/index.ts

```diff
@@ -1,16 +1,34 @@
 import { initializeGraphQLClient } from '../api';
 import { InitialzeOptions } from '../interfaces';
 import { refreshStore } from '../store';
+import { InitilizationError } from '../errors';
+import { validateConfiguration } from './validation';
 
 export let _configuration: InitialzeOptions;
 
+const startPolling = (configuration: InitialzeOptions) => {
+  const { pollingEnabled, pollIntervalSeconds } = configuration;
+
+  if (pollingEnabled) {
+    setInterval(() => {
+      refreshStore().catch((err) => {
+        // TODO error handle
+      });
+    }, pollIntervalSeconds * 1000);
+  }
+};
+
 export const initialize = async (options: InitialzeOptions) => {
   try {
+    validateConfiguration(options);
     _configuration = Object.freeze(options);
 
     initializeGraphQLClient(_configuration);
     await refreshStore();
+    startPolling(_configuration);
   } catch (err) {
-    // TODO error handle
+    if (err instanceof InitilizationError) {
+      throw err;
```

Comment:
think this is ok? That the SDK throws an error if its configured incorrectly?

(There will be no other runtime errors thrown)